cellengine/utils/converter.py

```diff
@@ -8,12 +9,6 @@
 def get_converter():
     converter = GenConverter()
 
-    def to_camel_case(snake_str: str) -> str:
```

Comment:
Please squash this commit (`src(): move to_camel_case to helpers`) with the commit where this method was added to reduce churn.

Reply:
:+1: